packages/generate-persisted-query-manifest/src/index.ts

```diff
@@ -101,6 +117,19 @@ const ERROR_MESSAGES = {
       name,
     )}" already defined in: ${COLORS.filepath(source.file.path)}`;
   },
+  uniqueOperationId: (
+    id: string,
+    operationName: string,
+    definedOperationName: string,
+  ) => {
+    return `\`createOperationId\` created an ID (${COLORS.identifier(
+      id,
+    )}) for operation named "${COLORS.name(
+      operationName,
+    )}" that has already been used for operation named "${COLORS.name(
+      definedOperationName,
+    )}".`;
+  },
 };
 
 function addError(source: DocumentSource, message: string) {
```

Comment:
```suggestion
function addError(
  source: Pick<DocumentSource, "file"> & Partial<DocumentSource>,
  message: string,
) {
```

Reply:
This way we can use `addError` further down without having to essentially duplicate it's logic.

Reply:
I like this. Updated in [f5638a1](https://github.com/apollographql/apollo-utils/pull/309/commits/f5638a1beec1ef04530ea6ac4b4cd0d083eed17e)